CHANGELOG.md

```diff
@@ -12,6 +12,13 @@
 * Fixed a bug where **doc-review** command failed on existing templates.
 * Fixed a bug where **validate** command failed when the word demisto is in the repo README file.
 * Added support for adding test-playbooks to the zip file result in *create-content-artifacts* command for marketplacev2.
+* Refactored the **format** command on update *fromversion*:
+  - Fixed a bug when changing content item, the *fromversion* field should remain.
+  - Updated the general default *fromversion* and the default *fromversion* of new content items (e.g. `Lists, Jobs..`).
+  - Removed the interactive in *update_playbook*.
```

Comment:
```suggestion
  - Removed the interactive option in *update_playbook*.
```

Reply:
why is it removed, by the way?

Reply:
The logic is compromised when receiving input from the user at this current stage of updating the fromVersion field. In addition, an option of interactivity has been added to all content types as can be seen in the following line.